reconcile/utils/oc.py

```diff
@@ -108,8 +108,12 @@ def wrapper(*args, **kwargs):
                 return result
 
             function_name = f'{function.__module__}.{function.__qualname__}'
-            ignore_reconcile_time = \
+            ignore_reconcile_time_annotation_present = \
                 annotations.get('qontract.ignore_reconcile_time') == 'true'
+            is_e2e_test = running_state.get('e2e_test', None)
```

Comment:
what about 

```suggestion
            is_e2e_test = running_state.get('e2e_test', None) is not None
```

to make the intent 100% clear?

Reply:
lgtm!